app/models/vm_or_template.rb

```diff
@@ -846,7 +846,8 @@ def ems_host_list
     [ext_management_system, "ems", host, "host"].each_slice(2) do |ems, type|
       if ems
         params[type] = {
-          :address    => ems.address,
+          # TODO: (Julian) Removing instances of address. Hopfully this won't be missed.
```

Comment:
Yes, just a few lines below that:

 miqVim = MiqVim.new(miqVimHost[:address], miqVimHost[:username], password_decrypt)